test-cypress/integration/setup/namespaces.spec.js

```diff
@@ -0,0 +1,352 @@
+
+        // Should provide pagination options
+        getNamespacesPerPageMenu().within(() => {
+            // Should show all namespaces by default (they are only 6 so they can be visualized all at once)
+            cy.get('.dropdown-toggle')
+                .should('contain', 'All')
+                .click();
+            cy.get('.page-size-option')
+                .should('have.length', 1)
+                .and('contain', 'All');
+            // Close the menu to avoid overlapping other elements
+            cy.get('.dropdown-toggle').click();
+        });
+
+        // Should show summary of results
+        getNamespacesHeaderPaginationInfo()
+            .should('be.visible')
+            .and('contain', 'Showing 1 - 6 of 6 results');
+
+        // Both header & footer pagination must be the same
+        getNamespacesHeaderPagination()
+            .should('be.visible')
+            .find('li')
+            // Single page + First & Last buttons
+            .should('have.length', 3);
+        getNamespacesPagination()
+            .should('be.visible')
+            .find('li')
+            // Single page + First & Last buttons
+            .should('have.length', 3);
+
+        // Verify default namespaces are present and interactable
+        const defaultPrefixes = Object.keys(DEFAULT_NAMESPACES);
+        getNamespaces().each(($row, $index) => {
+            const expectedPrefix = defaultPrefixes[$index];
+            const expectedNamespace = DEFAULT_NAMESPACES[expectedPrefix];
+
+            cy.wrap($row)
+                .find('.namespace-prefix')
+                .should('contain', expectedPrefix);
+
+            cy.wrap($row)
+                .find('.namespaceURI')
+                .should('contain', expectedNamespace);
+
+            cy.wrap($row)
+                .find('.select-namespace')
+                .should('be.visible')
+                .and('not.be.disabled');
+
+            cy.wrap($row)
+                .find('.edit-namespace-btn')
+                .should('be.visible')
+                .and('not.be.disabled');
+
+            cy.wrap($row)
+                .find('.delete-namespace-btn')
+                .should('be.visible')
+                .and('not.be.disabled');
+        });
+    });
+
+    it('should filter existing namespaces', () => {
+        getNamespacesFilterField()
+            .should('have.value', '')
+            .type('owl')
+            .should('have.value', 'owl');
+        getNamespaces()
+            .should('have.length', 1)
+            .and('contain', DEFAULT_NAMESPACES['owl']);
+        getNamespacesHeaderPaginationInfo()
+            .should('be.visible')
+            .and('contain', 'Showing 1 - 1 of 1 results');
+
+        getNamespacesFilterField()
+            .clear()
+            .type('missing_prefix');
+        // The table should still be visible but without any results
+        getNamespacesTable().should('be.visible');
```

Comment:
This test will fail until the GDB-3817 issue is fixed

Reply:
Fixed. See: https://github.com/Ontotext-AD/graphdb-workbench/pull/51